stock_warehouse_flow/models/stock_location_route.py

```diff
@@ -17,6 +17,7 @@ class StockLocationRoute(models.Model):
         compute="_compute_applicable_flow_ids",
         string="Applicable Flows",
     )
+    disable_warehouse_flow_on_confirm = fields.Boolean()
```

Comment:
Why not a selection field instead:
```suggestion
    apply_flow_on = fields.Selection(selection=[("on_confirm", "When move is confirmed")])
```
And the glue module between flow & release will add a new value:
```python
apply_flow_on = fields.Selection(selection_add=[("on_release", "When move is released")])
```
And code to adapt to check this option.

Reply:
done